python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -251,8 +252,16 @@ def assert_any_corrections(self, data):
         AssertionError
             If one of more attribute of the topic is set to True.
         """
+        corrections = []
+        for corr in self.valid_corrections:
+            if hasattr(data, corr):
+                corrections.append(corr)
```

Comment:
So, the idea is that this method can receive both `disableCorrection` or `enableCorrection` and they might have different names for some corrections, e.g. `disableAll` or `enableAll`. And I store all available corrections in a single array. I could have them separated but then I would need to check which topic I'm passing here. I did implemented it slightly different now, though. Following your suggestion furthermore, it will now do:

```python
assert any([getattr(data, corr, False)
                    for corr in self.corrections]), \
            "At least one correction must be set."
```